.github/workflows/_linux-test.yml

```diff
@@ -114,22 +120,22 @@ jobs:
         with:
           docker-image: ${{ steps.calculate-docker-image.outputs.docker-image }}
 
-      - name: Check if in a ARC runner
+      - name: Check if in a container runner
         shell: bash
         id: check_arc_runner
-        run: echo "IN_ARC_RUNNER=$([ -f /.inarc ] && echo true || echo false)" >> "$GITHUB_OUTPUT"
+        run: echo "IN_CONTAINER_RUNNER=$([ -f /.inarc ] && echo true || echo false)" >> "$GITHUB_OUTPUT"
 
       - name: Install nvidia driver, nvidia-docker runtime, set GPU_FLAG
         id: install-nvidia-driver
         uses: pytorch/test-infra/.github/actions/setup-nvidia@main
-        if: ${{ contains(inputs.build-environment, 'cuda') && !contains(matrix.config, 'nogpu') && steps.check_arc_runner.outputs.IN_ARC_RUNNER == 'false' }}
+        if: ${{ contains(inputs.build-environment, 'cuda') && !contains(matrix.config, 'nogpu') && steps.check_arc_runner.outputs.IN_CONTAINER_RUNNER == 'false' }}
```

Comment:
Note that setup-nivdia not only install the driver, but also sets `GPU_FLAG` https://github.com/pytorch/test-infra/blob/main/.github/actions/setup-nvidia/action.yml#L246.  If this step is skipped in container runner, we will need to have to set `GPU_FLAG` here